examples/multiple_data_source_examples/public/components/home.tsx

```diff
@@ -0,0 +1,266 @@
+          id: 'home',
+          items: [...navItems],
+        },
+      ]}
+    />
+  );
+});
+
+export const Home = ({
+  basename,
+  notifications,
+  savedObjects,
+  dataSourceEnabled,
+  setActionMenu,
+  dataSourceManagement,
+  navigateToApp,
+  navigation,
+}: HomeProps) => {
+  const pages = [
+    {
+      title: 'Data Source Selector',
+      id: 'selector',
+      component: (
+        <DataSourceSelectorExample
+          savedObjects={savedObjects}
+          notifications={notifications}
+          dataSourceEnabled={dataSourceEnabled}
+          dataSourceManagement={dataSourceManagement}
+        />
+      ),
+    },
+    {
+      title: 'Data Source Multi Selectable',
+      id: 'multi_selectable',
+      component: (
+        <DataSourceMultiSelectableExample
+          savedObjects={savedObjects}
+          notifications={notifications}
+          dataSourceEnabled={dataSourceEnabled}
+          setActionMenu={setActionMenu}
+          dataSourceManagement={dataSourceManagement}
+        />
+      ),
+    },
+    {
+      title: 'Data Source Selectable',
+      id: 'single_selectable',
+      component: (
+        <DataSourceSelectableExample
+          savedObjects={savedObjects}
+          notifications={notifications}
+          dataSourceEnabled={dataSourceEnabled}
+          setActionMenu={setActionMenu}
+          dataSourceManagement={dataSourceManagement}
+        />
+      ),
+    },
+    {
+      title: 'Data Source Readonly View',
+      id: 'single_readonly',
+      component: (
+        <DataSourceViewExample
+          savedObjects={savedObjects}
+          notifications={notifications}
+          dataSourceEnabled={dataSourceEnabled}
+          setActionMenu={setActionMenu}
+          dataSourceManagement={dataSourceManagement}
+        />
+      ),
+    },
+    {
+      title: 'Data Source List All',
+      id: 'list_all',
+      component: (
+        <DataSourceListAllExample
+          savedObjects={savedObjects}
+          notifications={notifications}
+          dataSourceEnabled={dataSourceEnabled}
+          setActionMenu={setActionMenu}
+          dataSourceManagement={dataSourceManagement}
+        />
+      ),
+    },
+    {
+      title: 'Data Source List Active',
+      id: 'list_active',
+      component: (
+        <DataSourceListActiveExample
+          savedObjects={savedObjects}
+          notifications={notifications}
+          dataSourceEnabled={dataSourceEnabled}
+          setActionMenu={setActionMenu}
+          dataSourceManagement={dataSourceManagement}
+        />
+      ),
+    },
+    {
+      title: 'Data Source View Via Top Nav Menu',
+      id: 'top_nav_menu',
+      component: (
+        <DataSourceViaTopNavMenuExample
+          savedObjects={savedObjects}
+          notifications={notifications}
+          dataSourceEnabled={dataSourceEnabled}
+          setActionMenu={setActionMenu}
+          navigation={navigation}
+        />
+      ),
+    },
+  ];
+  return (
```

Comment:
Is it possible to make route more easy?

```
        {pages.map(({ id, component }) => (
          <Route key={id} path={`/${id}`} render={() => component} />
        ))}

```

Reply:
Right, refactored this to loop through to construct the route